doc/source/release-notes/Percona-Server-5.7.33-36.rst

```diff
@@ -0,0 +1,34 @@
+.. _PS-5.7.33-36:
+
+================================================================================
+*Percona Server for MySQL* 5.7.33-36
+================================================================================
+
+:Date: March 1, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.33-36
+includes all the features and bug fixes available in
+`MySQL 5.7.33 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-33.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+New Features
+================================================================================
+
+* :jirabug:`PS-5364`: Update Hashicorp Vault plugin to use v2 API (Thanks to user aprokofyev for reporting this issue)
+* :jirabug:`PS-7447`: Backport variable innodb_buffer_pool_in_core_file and processing (Upstream :mysqlbug:`101825`)
+* :jirabug:`PS-7459`: Backport of InnoDB: Group purging of rows by table ID (`WL#9387 <https://dev.mysql.com/worklog/task/?id=9387>`_) to PS 5.7.32
+
```

Comment:
shouldn't this be 5.7.33-36? this might be wrong entry in JIRA and should be changed there

Reply:
I believe the latest version was 5.7.32 when the issue was created. So I think we can change it to PS 5.7.